core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sink/SimpleSinkOpExecConfig.scala

```diff
@@ -13,18 +10,20 @@ import edu.uci.ics.amber.engine.common.virtualidentity.{
   OperatorIdentity
 }
 import edu.uci.ics.amber.engine.operators.SinkOpExecConfig
+import edu.uci.ics.texera.workflow.common.IncrementalOutputMode
 import edu.uci.ics.texera.workflow.common.tuple.schema.OperatorSchemaInfo
 
-import scala.collection.mutable
-import scala.concurrent.ExecutionContext
-
-class SimpleSinkOpExecConfig(tag: OperatorIdentity, val operatorSchemaInfo: OperatorSchemaInfo)
-    extends SinkOpExecConfig(tag) {
+class SimpleSinkOpExecConfig(
+    tag: OperatorIdentity,
+    val operatorSchemaInfo: OperatorSchemaInfo,
```

Comment:
a minor comment, can this config file be unified with existing common configs?

Reply:
yes, this is the intention, we'll do this refactoring later.